staging/src/k8s.io/component-base/metrics/prometheus/slis/routes.go

```diff
@@ -38,8 +38,8 @@ type SLIMetrics struct{}
 func (s SLIMetrics) Install(m mux) {
 	installOnce.Do(func() {
 		Register(Registry)
-		m.Handle("/metrics/slis", metrics.HandlerFor(Registry, metrics.HandlerOpts{}))
 	})
+	m.Handle("/metrics/slis", metrics.HandlerFor(Registry, metrics.HandlerOpts{}))
```

Comment:
qq: will it expose the same set of metrics from two different kubelet ports, after this change?

Reply:
yes I believe so. The metrics still get registered once for both servers via Registry() which still happens under `installOnce.Do()`.
I also did a local check to verify it against a GKE cluster and its exposing the same metrics in the 4-5 times I restarted kubelet and looked at the endpoint for both ports.